playbooks/katello/katello_pulp3_installation.yaml

```diff
@@ -0,0 +1,119 @@
+---
```

Comment:
Can we use your https://github.com/redhat-performance/satperf/blob/master/playbooks/katello/katello_nightly_installation.yaml instead of this?

Reply:
ack we can use this.